CODESTYLEGUIDE.md

```diff
@@ -95,24 +95,53 @@ We want to keep a standard code structure that is based on the original Reusable
 
     // Building block definitions
     // API definitions ordered alphabetically
-  } 
+  }
 
 ```
 
 We like to do some light 'data cleaning' inside the `cleanData` method. That would usually include making sure that numbers are not strings. We also tend to use a reduce function, so we will create a copy of the original data and cast only the usual quantities or values we need for creating the chart.
 
-Note that most of the building blocks are dependant on the type of chart. For example, on the donut chart, we won't need to create axis or draw them, so those blocks won't be there.
+Note that most of the building blocks depend on the chart type. For example, on the donut chart, we won't need to create axis or draw them, so those blocks won't be there.
 
 
 **[⬆ back to top](#table-of-contents)**
 
 ## Comments
-We enforce jsDoc comments in our review process, so all of the functions will need a comment. We need also to be mindful and when we change the code of a function, check if its comments need an update.
+We enforce jsDoc comments in our review process, so all the functions will need a comment. We need also to be mindful and when we change the code of a function, check if its comments need an update.
 
 This is really important as our documentation gets generated from these jsDoc comments. You can read more about it [on the Contributing document](https://github.com/eventbrite/britecharts/blob/master/.github/CONTRIBUTING.md#jsdoc).
 
 **[⬆ back to top](#table-of-contents)**
 
+## Append Indentation
+It is a [usual pattern](https://bost.ocks.org/mike/d3/workshop/#35) in the D3.js world to apply a slightly different indentation to operations that change the selection, like the 'apply' method of a selection. We will apply two spaces instead of four:
```

Comment:
Maybe "when the selection changes, we indent with two spaces instead of four". Something to make it more clear that usually we indent with 4 spaces, but in this case we use 2.